src/fireblocks-sdk.ts

```diff
@@ -1159,6 +1161,13 @@ export class FireblocksSDK {
         return await this.apiClient.issueGetRequest(url);
     }
 
+    /**
+     * Get addresses for a vault account for asset with paging
+     */
+        public async getPaginatedAddresses(accountId: string, assetId: string, paginatedAddresseRequestFilters?: OptionalPaginatedAddresseRequestFilters): Promise<PaginatedAddressesResponse> {
```

Comment:
I believe you named this arg in our yaml spec (Doc) as vaultAccountId, so better align on this here